.clang-tidy

```diff
@@ -0,0 +1,44 @@
+---
+Checks:          '-*,bugprone-*,cppcoreguidelines-*,modernize-*,concurrency-*'
+WarningsAsErrors: ''
+HeaderFilterRegex: ''
+AnalyzeTemporaryDtors: false
+FormatStyle:     file
+CheckOptions:
+  - key:             cert-dcl16-c.NewSuffixes
```

Comment:
this options doesn't seem necessary since the check isn't run?

Reply:
AnalyzeTemporaryDtors is an artifact from an older clang-tidy version, notably 6 and below. Yeah it was dumped while dumping the config. 
FormatStyle: use the .clang-format file when its set to file. So, I guess that's okay to keep.